bokeh/client/session.py

```diff
@@ -164,10 +165,7 @@ def show_session(session_id=None, url='default', app_path='/',
             server_url = coords.server_url
             session_id = coords.session_id
 
-        if controller is None:
-            from bokeh.util.browser import get_browser_controller
```

Comment:
We don't normally defer imports. The fact that it was done here makes me suspect there is a reason, though can't say offhand what it would've been. Wouldn't the change on line 293 be sufficient in any case? I think if you pass along `browser` as you have done there, then `show_session` will work without any changes? I can't say for certain there are no places that rely on being able to pass in a controller explicitly, so I am reticent to make a breaking change here.